ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/traffic-policies.j2

```diff
@@ -149,6 +149,36 @@ No Match condition configured.
 {%             endif %}
 
 {%         endfor %}
+{%     endif %}
+{# Traffic-Policy Interfaces #}
+{%     set traffic_policy_interfaces = [] %}
+{%     for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
```

Comment:
please change this line to `for ethernet_interface in ethernet_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name')`
This will make the `ethernet_interface` var into a dict, so adjust below as needed.
Implement the same for port-channels.
This means the traffic_policy_interfaces will be a list of dicts with mixed port-channels and ethernet, so can just run over this single list and don't mind which type, since the `traffic_policy` stuff is the same.

Reply:
Implemented the suggested stuff.